Swift/Adhan/Adhan.swift

```diff
@@ -403,6 +408,35 @@ public struct Qibla {
     }
 }
 
+/* Sunnah times for a location and date using the given prayer times.
+All prayer times are in UTC and should be display using an DateFormatter that
+has the correct timezone set. */
+public struct SunnahTimes {
+    public let midnight: Date
+    public let lastThird: Date
```

Comment:
for clarity perhaps "lastThirdOfTheNight", I know it's a bit longer but I think it's better to be more clear.